roles/kubernetes/master/tasks/post-upgrade.yml

```diff
@@ -6,6 +6,7 @@
   delegate_to: "{{item}}"
   with_items: "{{groups['kube-master']}}"
   when: needs_etcd_migration|bool
+  run_once: true
```

Comment:
This task will run once for list {groups['kube-master']}} by first host in group. Without this run_once every host in play will run task for list.